huaweicloud/provider.go

```diff
@@ -585,6 +584,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_vbs_backup_policy":               resourceVBSBackupPolicyV2(),
 			"huaweicloud_vpc_bandwidth":                   eip.ResourceVpcBandWidthV2(),
 			"huaweicloud_vpc_eip":                         eip.ResourceVpcEIPV1(),
+			"huaweicloud_eip_associate":                   eip.ResourceEIPAssociate(),
```

Comment:
do you plan to get rid of the `vpc` from `huaweicloud_vpc_eip` and `huaweicloud_vpc_bandwidth`?

Reply:
not yet, I think we can keep it as it is.

I will add a new resource named **huaweicloud_bandwidth_associate**, so I get rid of vpc in this resource